Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.1.4'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
```

Comment:
We'll probably want `mysql2` but this can be a separate issue.

Reply:
https://github.com/GraphQL-Query-Planner/webscale/issues/4